### ISSUE TITLE
Please add an ALT button and the possibility to lock the function keys

### ISSUE THREAD

**Issue by rafal-1990** [ish-app/ish#1559]
I don't use a hardware keyboard and luckily there are buttons for `Tab`, `Ctrl`, `Esc` and arrow keys.  
But there is no button for `Alt` at all 😞. 
It would be great if an `Alt` button could be added. 
And an option to hold the function buttons (e.g. by tapping them twice in quick succession) would be also great.

**Comment by saagarjha** [comment]
What would you like the alt key to do? On iOS I guess it lets you type special characters, but I'm not sure if that's what you want it for…

**Comment by rafal-1990** [comment]
There are some keyboard shortcuts with `alt`  in bash (see <https://ss64.com/bash/syntax-keyboard.html>), which I want to use.
But also in some other programs like vim or tmux alt can be helpful to create abbreviations.

**Comment by saagarjha** [comment]
Those are really meta-key shortcuts, which you can get by hitting escape and then the combination key.

**Comment by rafal-1990** [comment]
That's right. By hitting `Esc` and than hitting another key I can enter a meta-key shortcut. 
But it is not possible to hold `Esc` and then hitting e. g. `f` multiple times to move fast forward one word in bash. 
You must press and release `Esc` before pressing the combination key. (This is not an issue of iSH. This works the same way on the PC, which is why I use the ALT key there and would like to have it in iSH as well.)

**Comment by Ctl-F** [comment]
The Nano text editor has certain features that are enabled using the 'alt' key.
